src/screens/Ticketing/Ticket/Details/DetailsScreen.tsx

```diff
@@ -41,6 +41,12 @@ export default function DetailsScreen({navigation, route}: Props) {
       orderVersion: fc.order_version,
     });
 
+  const onInspectionNavigate = () =>
+    fc &&
+    navigation.push('TicketInspection', {
+      orderId: fc.order_id,
```

Comment:
Couldn't we just pass the entire `fc` object here? and same for navigating to 'TicketReceipt'?

Reply:
We could, but that would make it more difficult to construct deep-links later to screens in app. We prefer to pass id's or simple serializable values when navigating. In this concrete example I'd also imagine that opening this screen when getting a push notification about a Bluetooth-challenge from an inspector is a plausible functionality.